integration/src/test/java/edu/unc/lib/boxc/integration/web/access/SearchEndpointIT.java

```diff
@@ -297,6 +299,29 @@ public void testSearchPublicUserWithMetadataOnlyPermissionCanSeeMetadata() throw
         }
     }
 
+    @Test
+    public void testDateDepositedInclusiveSearch() throws Exception {
+        createDefaultObjects();
+        var currentYear = Year.now().getValue();
+        assertResultCountEquals(SEARCH_URL + "/?added=" + currentYear + ",", 5);
+        assertResultCountEquals(SEARCH_URL + "/?added=" + currentYear + "," + currentYear, 5);
+        assertResultCountEquals(SEARCH_URL + "/?added=," + currentYear, 5);
+        assertResultCountEquals(SEARCH_URL + "/?added=" + (currentYear + 1) + ",", 0);
+    }
+
+    private void assertResultCountEquals(String url, int expectedCount) throws IOException {
+        assertResultCountEquals(new HttpGet(url), expectedCount);
+    }
+
+    private void assertResultCountEquals(HttpGet getMethod, int expectedCount) throws IOException {
```

Comment:
can you put this in the parent class where i have the other custom assert methods?